.github/workflows/main.yml

```diff
@@ -36,13 +34,13 @@ jobs:
         run: |
           mkdir build-${{matrix.TLS}}-${{matrix.CC}}
           cd build-${{matrix.TLS}}-${{matrix.CC}}
-          $GITHUB_WORKSPACE/configure --disable-documentation --enable-examples --enable-tests --disable-dtls CC=${{matrix.CC}}
+          $GITHUB_WORKSPACE/configure --disable-silent-rules --disable-documentation --enable-examples --enable-tests --disable-dtls CC=${{matrix.CC}}
```

Comment:
I did not disable silent compilation when switching to GH workflows as it has turned out that the full output was not required for trouble shooting for a long time (at least by me).
Feel free to leave it or not. (Also in line 43.)

Reply:
I had put it in to troubleshoot #606 as it was compiling locally but failing with GH workflows.  I had not appreciated my local build was using /usr/local/include/coap2 which is not there when building for the first time.  I think I would prefer to leave it in for now.